doc/building.md

```diff
@@ -26,10 +26,10 @@ Dependencies are needed for:
 
 Dependency list:
 
-    C     gcc >=4.9 or clang >=3.4
-    CRA   python >=3.4
+    C     gcc >= 4.9 or clang >= 3.4 (clang >= 3.5 for Mac OS X)
+    CRA   python >= 3.4
     C     cython >= 0.22
-    C     cmake >=3.0.0
+    C     cmake >= 3.0.0
       A   numpy
       A   python imaging library (PIL) -> pillow
     CR    opengl >=2.1
```

Comment:
if you're adding all the spaces after the `>=` then please do it everywhere :)
(the `>=0.0.0` without space syntax is from gentoo version numbering btw, that's why I actually wanna keep it)